api/v1beta2/foundationdbcluster_types.go

```diff
@@ -964,10 +964,14 @@ func GetProcessGroupConditionType(processGroupConditionType string) (ProcessGrou
 		return MissingService, nil
 	case "MissingProcesses":
 		return MissingProcesses, nil
+	case "ResourcesTerminating":
+		return ResourcesTerminating, nil
 	case "SidecarUnreachable":
 		return SidecarUnreachable, nil
 	case "PodPending":
 		return PodPending, nil
+	case "Ready":
```

Comment:
The ready condition is never set, I believe that's the reason why it's absent here but it doesn't hurt :)Just wanted to give you the context here.

Reply:
I saw that! but it did make sense to me to have all the conditions here